### ISSUE TITLE
UserWarning in InitalCondition.py

### ISSUE THREAD

**Issue by maxxxzdn** [ComputationalRadiationPhysics/NeuralSolvers#20]
https://github.com/ComputationalRadiationPhysics/NeuralSolvers/blob/2a9457257b35265d219db245eb765c6132559748/PINNFramework/InitalCondition.py#L28

/home/maxxxzdn/.local/lib/python3.8/site-packages/torch/nn/modules/loss.py:446: UserWarning: Using a target size (torch.Size([batch_size])) that is different to the input size (torch.Size([batch_size, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)

Using `prediction = model(x).reshape(-1)` solves the problem

**Comment by StillerPatrick** [comment]
could you say me which was the shape of x in the initial condition ?

**Comment by maxxxzdn** [comment]
It was [batch_size,3]

**Comment by StillerPatrick** [comment]
but looks correct me ?

**Comment by StillerPatrick** [comment]
and the shape of the predictions ?

**Comment by maxxxzdn** [comment]
[batch_size, 1]

Perhaps it worked fine, but I got this annoying UserWarning that [batch_size] != [batch_size,1]

**Comment by StillerPatrick** [comment]
ok :) if you shape your data correctly which is always batch size one, it works fine :)

**Comment by maxxxzdn** [comment]
I see. I will reshape my output tensor in the dataset then. 
Probably it makes sense to write some instructions for a dataset structure later on
